assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/buttons/taxes/TaxesButtons.tsx

```diff
@@ -18,7 +18,7 @@ const TaxesButtons: React.FC = () => {
 	// since we load all the default prices in EDTR
 	// so if the the number of TPC taxes is equal to number of all default taxes
 	// It means that TPC has all the possible tax prices
-	if (defaultTaxPrices.length === tpcDefaultTaxPrices.length) {
+	if (tpcDefaultTaxPrices.length && defaultTaxPrices.length === tpcDefaultTaxPrices.length) {
```

Comment:
`tpcDefaultTaxPrices` won't be `undefined` or `null` ?

Reply:
No, it comes from predicate as array